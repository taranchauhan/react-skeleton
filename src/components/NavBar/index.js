import React from 'react';
import styled from 'styled-components';

import logo from './logo.png';

const Header = styled.header`
  display: block;
  background: #205061;
  position: relative;
  z-index: 99999;
  top: 0;
  width: 100%;
  height: 106px;
`;

const Container = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: center;
  width: 80%;
  z-index: 99;
  height: 100%;
  max-width: 1080px;
  margin: auto;
  text-align: left;
`;

const Logo = styled.img`
  width: 100px;
`;

const App = () => (
  <Header>
    <Container>
      <Logo src={logo} alt="Logo" />
    </Container>
  </Header>
);

export default App;
