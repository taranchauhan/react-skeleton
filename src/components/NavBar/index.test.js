import React from 'react';
import { shallow } from 'enzyme';
import NavBar from './index';

describe('<NavBar />', () => {
  it('should render the NavBar', () => {
    const wrapper = shallow(<NavBar />);
    expect(wrapper).toMatchSnapshot();
  });
});
