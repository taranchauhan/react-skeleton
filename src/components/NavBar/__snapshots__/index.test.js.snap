// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavBar /> should render the NavBar 1`] = `
<styled.header>
  <styled.div>
    <styled.img
      alt="Logo"
      src="logo.png"
    />
  </styled.div>
</styled.header>
`;
